x-pack/plugins/observability_solution/logs_shared/server/services/log_views/log_views_client.test.ts

```diff
@@ -301,7 +301,7 @@ describe('LogViewsClient class', () => {
               "type": "keyword",
             },
           },
-          "scriptedFields": Array [],
+          "scriptedFields": Object {},
```

Comment:
Is it expected that type is different here now?

Reply:
@jughosta Yes, its a snapshot of the whole data view, including some private state.